pkg/cmd/pr/edit/edit.go

```diff
@@ -50,6 +50,7 @@ func NewCmdEdit(f *cmdutil.Factory, runF func(*EditOptions) error) *cobra.Comman
 		Short: "Edit a pull request",
 		Example: heredoc.Doc(`
 			$ gh pr edit 23 --title "I found a bug" --body "Nothing works"
+			$ gh pr edit 23 --base "base-branch-name" 
```

Comment:
I don't think we need to demonstrate every flag in the examples; just the more interesting or unusual ones. The `--base` flag is documented well enough by its description and needs no example.